tests/ui/closures/2229_closure_analysis/diagnostics/borrowck/borrowck-4.rs

```diff
@@ -16,5 +16,5 @@ fn foo () -> impl FnMut()->() {
 }
 fn main() {
     let c = foo();
-    c();
+    c(); //~ ERROR: not declared as mutable
```

Comment:
Should we just mark `c` as mutable?

Reply:
I was wondering about things like that... I can do that if preferred

Reply:
Same thing for missing `unsafe` blocks?